packages/syft/src/syft/core/tensor/autograd/backward_ops/transpose.py

```diff
@@ -12,7 +12,7 @@
 class TransposeOp(Op):
     """Repeat operation across a dimension"""
 
-    def forward(self, x: AutogradTensor, *dims: tuple) -> AutogradTensor:
+    def forward(self, x: AutogradTensor, *dims: tuple) -> AutogradTensor:  # type: ignore
```

Comment:
Should be Tuple[Type], what is the type in dims, is this just Tuple[int] shape?

Reply:
fixed